src/CSnakes.SourceGeneration/PythonStaticGenerator.cs

```diff
@@ -122,7 +139,14 @@ from f in functionNames
                         using (GIL.Acquire())
                         {
                             logger?.LogDebug("Importing module {ModuleName}", "{{fileName}}");
-                            module = Import.ImportModule("{{fileName}}");
+                            if (!string.IsNullOrEmpty(encodedSource))
+                            {
+                                module = Import.ImportModule("{{fileName}}", Encoding.UTF8.GetString(Convert.FromBase64String(encodedSource)), "{{fileName}}.py");
```

Comment:
What's the purpose of encoding the Python source into Base64? I'd be strongly inclined to embed the Python source in clear text, as-is.

Reply:
To preserve the encoding and not have issues with the encoding of the source file interfering with the original source. Also it takes up less space.